content/docs/idp/get-started/private-registry.md

```diff
@@ -22,38 +22,46 @@ Developers leverage templates and components in their preferred workflows, wheth
 
 If you're new to Pulumi components, the [Build a Component](/docs/iac/using-pulumi/extending-pulumi/build-a-component/) guide is a great resource for getting started. Once you've authored your component, push it to a GitHub or GitLab repository that Pulumi can access. Private repositories are supported, but the repository must be open to inbound requests.
 
-#### Component Versioning
-
-Pulumi checks for a git version tag when the `publish` command is executed and stores it as the component version. The tag must adhere to [to the semantic versioning standard](https://semver.org/).
+The `publish` CLI command is used to publish components to the private registry.
 
-#### Component README
+```bash
+pulumi package publish <provider|schema>
+```
 
-A README is required when publishing a component. Pulumi renders markdown README files in the private registry. They're a great way to provide context for a component. By default, the Pulumi CLI looks for a README in the component's root directory. The `--readme` flag can be used to specify a custom source.
+For example, if your github organization is ACME and you are publishing the `k8s-cluster` component, you'd run:
 
 ```bash
-pulumi package publish /path/to/your/component --readme README_LOCATION
+pulumi package publish git://github.com/acme/k8s-cluster
 ```
 
-#### Specifying an Organization
+#### Component Versioning
```

Comment:
I think we should point out that it's inferring the latest version when no version is provided

Reply:
I've moved this sentence up a little to make it easier to spot.